src/main/java/com/minecolonies/coremod/colony/buildings/modules/LivingBuildingModule.java

```diff
@@ -101,6 +102,12 @@ public int getModuleMax()
         return building.getBuildingLevel();
     }
 
+    @Override
+    public JobEntry getJobEntry()
+    {
+        return null;
```

Comment:
You sure this is safe like this?
I am pretty sure that method is marked as @NotNull and if so, I would rather this does something else....

Reply:
should probably extract the job assignment into a different interface, so we don't require a job on non-working assignment modules

Reply:
I added this in here to be able to match them without requiring 100 messages.

Reply:
hm not sure I see how this is connected to having 100 messages, care to elaborate?

Reply:
Basically, the jobEntry is used to decide which IAssignmentModule is called. If it's a job, it's one of the job assignment modules, else it's the livingBuildingModule. I.e. in the warehouse we got couriers. This allows to generalize the modules a little bit and spare us duplicate windows and messages (i.e. more if we create more cases)

Reply:
I tried to adjust this.